sdk/core/abort-controller/changelog.md

```diff
@@ -0,0 +1,3 @@
+Changelog
+1.0.0-preview.2 - 2019-09-09
+Listeners attached to an `AbortSignal` now receive an event with the type `abort`. (PR #4756)
```

Comment:
Wasnt this always the case?
I thought the change this time was related to `displatchEvent` and `libdom`

Reply:
This was not the case until the PR that's shown. I can also add that `dispatchEvent` was added but should not be called (it will throw an error).